@here/harp-mapview/test/PoiBatchRegistryTest.ts

```diff
@@ -25,12 +25,11 @@ describe("PoiBatchRegistry", () => {
         registry = new PoiBatchRegistry(renderer.capabilities);
     });
     describe("registerPoi", function () {
-        it("marks PoiInfo as invalid if it has no image item", () => {
+        it("return undefined if poiInfo has no image item", () => {
```

Comment:
force here that the poi is still valid.
Also, add some test to check that a poi is valid after a miss on findImageItem()

Reply:
Done